helfi_tpr.install

```diff
@@ -699,3 +699,24 @@ function helfi_tpr_update_8039() : void {
       ->installFieldStorageDefinition($name, 'tpr_unit', 'helfi_tpr', $field);
   }
 }
+
+/**
+ * Add name_synonyms field to Service entity.
+ */
+function helfi_tpr_update_8052() : void {
```

Comment:
update the id of the hook with 8040 (the next number after last existing update hook)